application-forum-ui/src/main/resources/ForumCode/Actions.xml

```diff
@@ -65,26 +67,34 @@
         $response.setStatus(403)
       #end
     #end
-  #elseif("$!request.action" == "addtopic")
+  #elseif("$!request.action" == 'addtopic')
     #if("$!request.title.trim()" != '')
       #if("$!request.space" != '')
-        #set($newTopicName = $request.page)
         #set($topicTitle = "$!request.title.trim()")
-        #set($topicDoc = $xwiki.getDocument($newTopicName))
+        #set($topicSpace = "$!request.space")
+        #if($supportsNS)
+          #set($topicSpaceReference = $services.model.resolveSpace($topicSpace))
```

Comment:
Nevermind the above. I was confused by the bad naming of `$topicSpace` and `$topicSpaceReference` (which should also be `$forumSpaceReference`).